dirsrvtests/tests/suites/clu/dsidm_group_test.py

```diff
@@ -0,0 +1,416 @@
+
+    log.info('Create test group')
+    if groups.exists(group_name):
+        test_group = groups.get(group_name)
+        test_group.delete()
+    else:
+        test_group = groups.create_test_group()
+
+    def fin():
+        log.info('Delete test group')
+        if test_group.exists():
+            test_group.delete()
+
+    request.addfinalizer(fin)
+
+
+@pytest.mark.skipif(ds_is_older("1.4.2"), reason="Not implemented")
+def test_dsidm_group_create(topology_st):
+    """ Test dsidm group create option
+
+    :id: 56e31cf5-0fbf-4693-b8d3-bc6f545d7734
+    :setup: Standalone instance
+    :steps:
+        1. Run dsidm group create
+        2. Check that a message is provided on creation
+        3. Check that created group exists
+    :expected results:
+        1. Success
+        2. Success
+        3. Success
+    """
+    
+    standalone = topology_st.standalone
+    group_name = 'test_group'
+    output = 'Successfully created {}'.format(group_name)
+
+    args = FakeArgs()
+    args.cn = group_name
+
+    log.info('Test dsidm group create')
+    create(standalone, DEFAULT_SUFFIX, topology_st.logcap.log, args)
+    check_value_in_log_and_reset(topology_st, check_value=output)
+
+    log.info('Check that group is present')
+    groups = Groups(standalone, DEFAULT_SUFFIX)
+    new_group = groups.get(group_name)
+    assert new_group.exists()
+
+    log.info('Clean up for next test')
+    new_group.delete()
+
+
+@pytest.mark.skipif(ds_is_older("1.4.2"), reason="Not implemented")
+def test_dsidm_group_delete(topology_st, create_test_group):
+    """ Test dsidm group delete option
+
+    :id: d285db5d-8efd-4775-b5f4-47bf27106da6
+    :setup: Standalone instance
+    :steps:
+        1. Run dsidm group delete on a created group
+        2. Check that a message is provided on deletion
+        3. Check that group does not exist
+    :expected results:
+        1. Success
+        2. Success
+        3. Success
+    """
+
+    standalone = topology_st.standalone
+    groups = Groups(standalone, DEFAULT_SUFFIX)
+    test_group = groups.get('test_group_2000')
+    output = 'Successfully deleted {}'.format(test_group.dn)
+
+    args = FakeArgs()
+    args.dn = test_group.dn
+
+    log.info('Test dsidm group delete')
+    delete(standalone, DEFAULT_SUFFIX, topology_st.logcap.log, args, warn=False)
+    check_value_in_log_and_reset(topology_st, check_value=output)
+
+    log.info('Check that group does not exist')
+    assert not test_group.exists()
+
+
+@pytest.mark.skipif(ds_is_older("1.4.2"), reason="Not implemented")
+def test_dsidm_group_modify(topology_st, create_test_group):
+    """ Test dsidm group modify option
+
+    :id: 43611907-b477-4793-9ed6-8cf2d43ddd6b
+    :setup: Standalone instance
+    :steps:
+        1. Run dsidm group modify add description value
+        2. Run dsidm group modify replace description value
+        3, Run dsidm group modify delete description value
```

Comment:
There needs to be "3. Run dsidm" with dot.

Reply:
Fixed, thank you